resource/inspire2019-prod/sidebarMetadata.json

```diff
@@ -32,17 +32,24 @@
         "subMetadata": []
       },
       {
-        "title": "Content area",
-        "layout": "",
-        "listings": "list-item",
-        "subtype": "contentArea",
+        "title": "Target Audience",
+        "layout": "no-link",
+        "listings": "",
+        "subtype": "targetAudience",
         "subMetadata": []
       },
       {
-        "title": "Products",
```

Comment:
Products need to stay, all of this needs to stay

Reply:
Restored.